.travis.yml

```diff
@@ -1,6 +1,4 @@
 sudo: false
 language: scala
-scala:
-    - 2.10.4
 script:
-    - sbt ++$TRAVIS_SCALA_VERSION test
\ No newline at end of file
+    - sbt +test
```

Comment:
`+test` will run the test target against all defined scala versions in build.sbt